samples/java_layout/src/main/java/io/customer/android/sample/java_layout/data/PreferencesDataStore.java

```diff
@@ -0,0 +1,55 @@
+package io.customer.android.sample.java_layout.data;
+
+
+import androidx.annotation.MainThread;
+import androidx.datastore.preferences.core.Preferences;
+import androidx.datastore.preferences.rxjava3.RxPreferenceDataStoreBuilder;
+import androidx.datastore.rxjava3.RxDataStore;
+
+import java.util.Map;
+
+import io.customer.android.sample.java_layout.SampleApplication;
+import io.reactivex.rxjava3.core.Flowable;
+
+public class PreferencesDataStore {
```

Comment:
It would be good to maybe create an `interface` for it, so in future if we are to add test cases we can mock do it easily.

Reply:
I don't think we ever need to setup unit tests for sample apps. This will only add complexity IMO. So I tried to keep things simple as the main goal for sample apps is to test integration with SDK.